### PR TITLE
Add release process doc

### PR DESCRIPTION
**Pull request by yannispanousis**

### CODE REVIEW

File: release-process/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+You should add a high-level description of the features in this release.
+
+You should label this release as a **pre-release** (tick the box at the bottom) until your PR is merged into production.
+
+- Once the PR is merged into production, visit your pre-release and hit **Publish**.
+
+# Merging any Hot Fixes back into master
+
+Because the hot fixes were done on the release branch and then went straight into production, we haven't got them yet in master. This means we need to bring them into master by **merging production into master**.
```
Comment (junkafarian) [inline release-process/README.md]: I think this should be standard practice as part of the release process regardless. We're probably going to have some release notes and a version bump as part of each release which we'll need back in the trunk.

Also hoping that we won't have much in the way of development going on in a non-feature branch.

**Comment by junkafarian** [comment]
very nice, let's look at what we can do from the command line to group up some of these release steps for consistency (eg. github releases track git tags and we might be able to extract the release notes from a `CHANGES` doc)

**Comment by yannispanousis** [comment]
@junkafarian CLI batching is definitely needed. We're hopefully going to iron this out a bit better today because on Friday we had an issue with putting the tag in the wrong branch

**Comment by ctolsen** [comment]
:+1:, we'll see how it works out later today :)

**Comment by yannispanousis** [comment]
Have updated it now if you want to take a quick look, otherwise we'll be going through it twice in about an hour from now.

File: release-process/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+- Open a PR for **release/v0.2.0** (head) into **production** (base)
+- Once the PR is merged into production, create a new release for that project via its release page, e.g. here is the [LWS release page](https://github.com/lystable/lws/releases).
+- Use tag **v0.2.0** with the **production** branch as the target branch.
+
+You should add a high-level description of the features in this release.
+
+Hit **Publish** at the bottom of the release page when done.
+
+# Merging any Hot Fixes back into master
```
Comment (junkafarian) [inline release-process/README.md]: looks good. let's drop the term hot fixes though

**Comment by junkafarian** [comment]
looks good with the wording change :+1:

**Comment by ctolsen** [comment]
Process worked well yesterday so think this is ready to go

**Comment by yannispanousis** [comment]
What shall we change 'Hot Fixes' to ? QA fixes? We would be diverging more from GitFlow terminology however.

**Comment by ctolsen** [comment]
Bug fixes? Hot fixes are only direct to production in git flow, I believe.

**Comment by yannispanousis** [comment]
@ctolsen you're correct, bug fixes it is

**Pull request merged**